src/services/agendaServices.js

```diff
@@ -1,178 +1,152 @@
 /**
- * name : common.js
+ * name : agendaServices.js
  * author : Vishnudas
  * created-date : 19-may-2022
- * Description : all common constants
-*/
+ * Description : agenda services.
+ */
 
 //Dependencies
-const Agenda = require('agenda');
-const configuration = require('../config');
-const needle = require('needle');
-const responseMessage = require('../generics/constants/responseMessage')
-const httpResponse = require('../generics/constants/httpResponse')
-const common = require('../generics/constants/common')
-
-const {sendErrorMail} = require('../generics/utils');
-const log = require("../models/log");
-
-//connect Agenda to default collection--agendaJobs
-const agenda = new Agenda({
-  db : {
-      address : configuration.mongourl,
-      collection : configuration.collection ? configuration.collection : undefined
-  }
-
-});
-
+const configuration = require('@root/config')
+const needle = require('needle')
+const responseMessage = require('@constants/responseMessage')
+const httpResponse = require('@constants/httpResponse')
+const common = require('@constants/common')
+const { sendErrorMail } = require('@generics/utils')
+const log = require('@models/log')
+const { agenda } = require('@configs/agenda')
 
 //restart agenda instances when server restarts
-const jobsReady = agenda._ready.then( async () => {
-  const jobDefCollection = agenda._mdb.collection( configuration.definition );
-
-  jobDefCollection.toArray = () => {
-      const jobsCursor = jobDefCollection.find();
-      return (jobsCursor.toArray).bind(jobsCursor)();
-  };
-  await jobDefCollection
-    .toArray()
-    .then((jobsArray) =>
-      Promise.all(jobsArray.map((job) => defineJob(job, jobDefCollection, agenda)))
-    );
-  await agenda.start();
-  return jobDefCollection;
-});
-
+const jobsReady = agenda._ready.then(async () => {
+	const jobDefCollection = agenda._mdb.collection(configuration.definition)
+
+	jobDefCollection.toArray = () => {
+		const jobsCursor = jobDefCollection.find()
```

Comment:
@VISHNUDAS-tunerlabs can we move all the queries to a single queries file for a model, refer https://github.com/ELEVATE-Project/mentoring/blob/master/src/db/entities/query.js

Reply:
Agenda does not have any model for data operations..So can skip this?